vector/src/main/java/im/vector/app/features/userdirectory/PendingSelection.kt

```diff
@@ -29,4 +29,11 @@ sealed class PendingSelection {
             is ThreePidPendingSelection -> threePid.value
         }
     }
+
+    fun getMxId(): String {
+        return when (this) {
+            is UserPendingSelection     -> user.userId
+            is ThreePidPendingSelection -> threePid.value
```

Comment:
this would be a mail or Pn but not a mxid

Reply:
Yes, but I think it will work, until the other user has not joined the room. @Florian14 do you confirm?